corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -41,6 +45,8 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
             self.programs = params.commtrack_programs;
             self.isShadowForm = params.isShadowForm;
 
+            self.saveOnlyEditedFormFieldsEnabled = toggles.toggleEnabled("SAVE_ONLY_EDITED_FORM_FIELDS");
```

Comment:
I'd just call `toggles.toggleEnabled("SAVE_ONLY_EDITED_FORM_FIELDS")` in the places where it's needed, rather than having an extra layer of code.